src/install-qtifw.ts

```diff
@@ -221,10 +221,44 @@ export async function installRequiredSystemDeps() {
           'libxcb-randr0-dev',
           'libxcb-shape0',
           'libxcb-cursor0',
-          'libdbus-1-3'
+          'libdbus-1-3',
+          'libwebp-dev'
         ],
         {silent: true}
       );
+      if (ARCH == 'arm64') {
+        if (!fs.existsSync('/lib/aarch64-linux-gnu/libwebp.so.6')) {
+          core.info(
+            'Linux arm64: libwebp.so from installer is outdated, symlink: ln -sf libwebp.so.7 /lib/aarch64-linux-gnu/libwebp.so.6'
+          );
+          // fs.symlinkSync('libwebp.so.7', '/lib/aarch64-linux-gnu/libwebp.so.6')
+          await exec.exec(
+            'sudo',
+            [
+              'ln',
+              '-sf',
+              'libwebp.so.7',
+              '/lib/aarch64-linux-gnu/libwebp.so.6'
+            ],
+            {silent: true}
+          );
+        }
+        if (!fs.existsSync('/lib/aarch64-linux-gnu/libtiff.so.5')) {
+          core.info(
+            'Linux arm64: libtiff.so from installer is outdated, symlink: ln -sf libtiff.so.6 /lib/aarch64-linux-gnu/libtiff.so.5'
+          );
+          await exec.exec(
+            'sudo',
+            [
+              'ln',
+              '-sf',
+              'libtiff.so.6',
+              '/lib/aarch64-linux-gnu/libtiff.so.5'
+            ],
+            {silent: true}
+          );
+        }
+      }
```

Comment:
And do some symlink shenanigans for linux arm64...

Reply:
These look like nice clever workarounds.

Reply:
Did you mean to add "incredibly brave" as well? :)